pyro/infer/__init__.py

```diff
@@ -23,13 +26,16 @@
     "is_validation_enabled",
     "ELBO",
     "EmpiricalMarginal",
+    "HMC",
     "Importance",
     "IMQSteinKernel",
     "infer_discrete",
     "JitTraceEnum_ELBO",
     "JitTraceGraph_ELBO",
     "JitTraceMeanField_ELBO",
     "JitTrace_ELBO",
+    "MCMC",
+    "NUTS",
```

Comment:
probably this is a long-awaited feature. :D I am not a fan of having different inference engines in different namespaces.